<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationViewer</class>
 <widget class="QWidget" name="OperationViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ops_buttons">
     <item>
      <widget class="QPushButton" name="add_op">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../picsou.qrc">
         <normaloff>:/resources/material-design/svg/cart-plus.svg</normaloff>:/resources/material-design/svg/cart-plus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_op">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="../../picsou.qrc">
         <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_op">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../../picsou.qrc">
         <normaloff>:/resources/material-design/svg/cart-off.svg</normaloff>:/resources/material-design/svg/cart-off.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_add_op">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/cart-plus.svg</normaloff>:/resources/material-design/svg/cart-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Add operation</string>
   </property>
  </action>
  <action name="action_edit_op">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/pencil.svg</normaloff>:/resources/material-design/svg/pencil.svg</iconset>
   </property>
   <property name="text">
    <string>Edit operation</string>
   </property>
  </action>
  <action name="action_remove_op">
   <property name="icon">
    <iconset resource="../../picsou.qrc">
     <normaloff>:/resources/material-design/svg/cart-off.svg</normaloff>:/resources/material-design/svg/cart-off.svg</iconset>
   </property>
   <property name="text">
    <string>Remove operation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../picsou.qrc"/>
 </resources>
 <connections/>
</ui>
